lib/rubocop/cop/rails/action_controller_flash_before_render.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Using `flash` assignment before `render` will persist the message for too long.
+      # Check https://guides.rubyonrails.org/action_controller_overview.html#flash-now
+      #
```

Comment:
If `SafeAutocorrect: false`, can you insert `@safety` section?
e.g. https://github.com/rubocop/rubocop-rails/blob/v2.15.2/lib/rubocop/cop/rails/arel_star.rb#L13-L17